dags/operators/postgres_to_s3_transfer.py

```diff
@@ -0,0 +1,90 @@
+from urllib.parse import urlparse
+import subprocess
+import logging
+import boto3
+
+import airflow.hooks
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+import utils.helpers as helpers
+
+
+class PostgresToS3Transfer(BaseOperator):
```

Comment:
Our second, actually https://github.com/opentrials/opentrials-airflow/blob/master/dags/operators/http_to_s3_transfer.py :smile:

We could split the operators, but it complicates things. Instead of a single task, we would need two tasks and a way of communication between them. On Airflow, each task can run on a different machine and on a different time, so the communication is more complicated than a simple pipe. They provide `XCom` to allow this communication (using the DB as an intermediary if I'm correct), so I don't think they're well suited for the few GBs of data we're transfering here.